towerlib/entities/schedule.py

```diff
@@ -0,0 +1,382 @@
+    @property
+    def name(self):
+        """The name of the schedule.
+
+        Returns:
+            string: The name of the schedule.
+
+        """
+        return self._data.get('name')
+
+    @property
+    def description(self):
+        """The description of the schedule.
+
+        Returns:
+            string: The description of the schedule.
+
+        """
+        return self._data.get('description')
+
+    @property
+    def extra_data(self):
+        """The extra data of the schedule.
+
+        Returns:
+            dict: The extra data of the schedule.
+
+        """
+        return self._data.get('extra_data')
+
+    @property
+    def inventory(self):
+        """The inventory this schedule belongs to.
+
+        Returns:
+            Inventory: The inventory this schedule belongs to.
+
+        """
+        return self._data.get('inventory')
+
+    @property
+    def scm_branch(self):
+        """The scm_branch of the schedule.
+
+        Returns:
+            string: The scm_branch of the schedule.
+
+        """
+        return self._data.get('scm_branch')
+
+    @property
+    def job_type(self):
+        """The job_type of the schedule.
+
+        Returns:
+            string: The job_type of the schedule.
+
+        """
+        return self._data.get('job_type')
+
+    @property
+    def job_tags(self):
+        """The job tags of the schedule.
+
+        Returns:
+            string: The job tags of the schedule.
+
+        """
+        return self._data.get('job_tags')
+
+    @property
+    def skip_tags(self):
+        """The tags to skip of the schedule.
+
+        Returns:
+            string: The tags to skip of the schedule.
+
+        """
+        return self._data.get('job_tags')
+
+    @property
+    def limit(self):
+        """The limit of the schedule.
+
+        Returns:
+            string: The limit of the schedule.
+
+        """
+        return self._data.get('limit')
+
+    @property
+    def diff_mode(self):
+        """Display mode for the execution the schedule.
+
+        Returns:
+            boolean: Are we displaying diff mode for the run?
+
+        """
+        return self._data.get('diff_mode')
+
+    @property
+    def verbosity(self):
+        """Verbosity of the run.
+
+        Returns:
+            string: Verbosity of the run .
+
+        """
+        return self._data.get('verbosity')
+
+    @property
+    def unified_job_template(self):
+        """Unified job template.
+
+        Returns:
+            JobTemplate: Unified job template .
+
+        """
+        url = self._data.get('related', {}).get('unified_job_template')
+        return self._tower._get_object_by_url('JobTemplate', url)  # pylint: disable=protected-access
+
+    @property
+    def enabled(self):
+        """Enables processing of this schedule.
+
+        Returns:
+            bool: Enables processing of this schedule.
+
+        """
+        return self._data.get('enabled')
+
+    @property
+    def dtstart(self):
```

Comment:
please use a friendlier variable name like datetime_start

Reply:
@SayantanKhanra10 already fixed this.